vnext/ReactWindowsCore/WinRTWebSocketResource.cpp

```diff
@@ -65,7 +68,7 @@ namespace
 
       void await_resume() const noexcept {}
 
-      void await_suspend(std::experimental::coroutine_handle<> resume)
+      void await_suspend(std::experimental::coroutine_handle<> resume) noexcept
```

Comment:
>noexcept [](start = 71, length = 8)

Please consider making all non-override methods noexcept, eg: WinRTWebSocketResource::WinRTWebSocketResource

Reply:
I'll open a GH issue for this.
After upgrading the boost version, I also want to make the interface methods noexcept.